app/controllers/tags_controller.rb

```diff
@@ -394,4 +392,18 @@ def tag_params
       associations_to_remove: []
     )
   end
+
+  def tag_search_params
+    params.require(:tag_search).permit(
+      :query,
+      :name,
+      :fandoms,
+      :type,
+      :fandom_ids,
```

Comment:
I don't think we need `fandom_ids` here or in tag_search_form's `ATTRIBUTES`, and the tests seem to pass without them. Am I missing something that's not covered by tests?

Reply:
You're right, I'll remove them.